tests/integration/test_integration_snapshots.py

```diff
@@ -322,3 +323,10 @@ def test_env_vars(use_ddtracerun, ddtrace_run_python_code_in_subprocess, run_pyt
 """,
         env=env,
     )
+
+
+@pytest.mark.snapshot(token="tests.integration.test_integration_snapshots.test_snapshot_skip")
+def test_snapshot_skip():
+    pytest.skip("Test that snapshot tests can be skipped")
```

Comment:
Shouldn't we not see a snapshot for this test if this test is skipped?

Reply:
Ah yeah I see the confusion. This repro isn't exactly the same as the issue that was being experienced. The case was when a test is skipped for one parametrization but not for another. So the snapshot is generated for the one run but skipped for the other. Previously this was failing.

Let me see if I can parametrize the test so that we can get closer to the real issue that was occurring.

Reply:
Not a blocker, if we can add a comment in the test explaining your comment above instead that is enough for me.